app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -144,10 +79,14 @@ def billed?
         end
 
         def doc_auth_success?
-          transaction_status_passed? &&
+          # really it's everything else excluding selfie
+          ((transaction_status_passed? &&
             true_id_product.present? &&
             product_status_passed? &&
             doc_auth_result_passed?
+           ) ||
+            attention_with_barcode?
+          ) && id_type_supported?
```

Comment:
I think that this is the core change in this PR? And the rest of the changes are mostly around code cleanliness?

Reply:
Would these changes be enough on their own to meet the A/Cs?

Reply:
@charleyf , no there are changes in `DocumentCaptureSession` or `DocumentCaptureSessionResult`